docker-app/qfieldcloud/core/cron.py

```diff
@@ -76,3 +77,27 @@ def do(self):
             },
             output="Job unexpectedly terminated.",
         )
+
+
+class DeleteDanglingProjectPackagesJob(CronJobBase):
+    schedule = Schedule(run_every_mins=60)
+    code = "qfieldcloud.delete_dangling_project_packages"
+
+    def do(self):
+        # get only the projects updated in the last a little more than an hour,
+        # as we assume the rest of the projects have been cleaned from dangling
+        # packages in the previous CRON run
+        projects = Project.objects.filter(
+            updated_at__gt=timezone.now() - timedelta(minutes=65)
```

Comment:
What about waiting a little before deleting packages, just to avoid deleting brand new packages (which could not yet be completely uploded, and which potentially another user just started downloading because for him the brand new package was not yet there ?)
```diff
      projects = Project.objects.filter(
-            updated_at__gt=timezone.now() - timedelta(minutes=65)
+            updated_at__gte=timezone.now() - timedelta(minutes=125),
+            updated_at__lt=timezone.now() - timedelta(minutes=65)
      )
```

Reply:
Well spotted. I do agree this is better.

Reply:
oops there should be a small overlap still:
```diff
      projects = Project.objects.filter(
-            updated_at__gt=timezone.now() - timedelta(minutes=65)
+            updated_at__gte=timezone.now() - timedelta(minutes=125),
+            updated_at__lt=timezone.now() - timedelta(minutes=60) # not 65
      )
```